src/style.css

```diff
@@ -1 +1,67 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
```

Comment:
You have two @font-face rules for the same font-family 'Avenir'. It's better to give them different names, for example 'Avenir' and 'Avenir-Heavy'. This way, you can control which one to use by changing the font-family.